extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -497,7 +497,7 @@ export class AmpAnalytics extends AMP.BaseElement {
    */
   generateRequests_() {
     if (!this.config_['requests']) {
-      if (!this.isInabox_) {
+      if (!this.isInabox_ || !this.isInFie_) {
```

Comment:
Please use the `allowParentPostMessage_` method instead.

Reply:
Done